src/stdx/prng.zig

```diff
@@ -333,16 +333,16 @@ test int {
     ));
 }
 
-fn test_bytes_T(T: type, want: Snap) !void {
+fn test_bytes_T(Int: type, want: Snap) !void {
```

Comment:
Should we also rename this function to `test_bytes_Int`?

Reply:
Good catch, done!